### ISSUE TITLE
Not working without rando

### ISSUE THREAD

**Issue by MerRM** [dpinela/Transcendence#1]
The mod will not work if Randomizer4 isn't installed and enabled even though Randomizer 4 is not a dependency for the mod
```
[ERROR]:[API] - Failed to load Mod `Transcendence`
[ERROR]:[API] - System.IO.FileNotFoundException: Could not load file or assembly 'RandomizerMod, Version=4.0.4.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
[ERROR]:[API] - File name: 'RandomizerMod, Version=4.0.4.0, Culture=neutral, PublicKeyToken=null'
[ERROR]:[API] -   at Modding.Mod.Initialize (System.Collections.Generic.Dictionary`2[TKey,TValue] preloadedObjects) [0x00000] in <2b54cdaf7a934f0eb297d50c4090ea9f>:0 
[ERROR]:[API] -   at Modding.ModLoader.LoadMod (Modding.ModLoader+ModInstance mod, System.Boolean updateModText, System.Collections.Generic.Dictionary`2[TKey,TValue] preloadedObjects) [0x00022] in <2b54cdaf7a934f0eb297d50c4090ea9f>:0 

```

**Comment by dpinela** [comment]
This is really baffling me - I'm checking if rando is installed before running the code that interacts with it (using `ModHooks.GetMod("Randomizer 4")`) and yet it still doesn't load. I suspect I'm missing something, because MapModS does something similar and it loads just fine in this situation.

**Comment by dpinela** [comment]
Fixed in version 1.0.1, which will be on Scarab soon™️. In the meantime you can grab it at https://github.com/dpinela/Transcendence/releases/tag/v1.0.1
